src/client/js/otp/config.js

```diff
@@ -313,7 +313,9 @@ otp.config.modes = {
     //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
     "CAR_PARK,WALK,TRANSIT"     : _tr('Park and Ride'),
     //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel Options widgets)
-    "CAR,WALK,TRANSIT"          : _tr('Kiss and Ride'),
+    "CAR_PICKUP,WALK,TRANSIT"   : _tr('Ride and Kiss (Car Pickup)'),
```

Comment:
Would it also be interesting to have an option for taxi here (i.e. car_pickup as a direct mode)?

Reply:
:+1: - added a `CAR_PICKUP,WALK` option to the client.